tools/maybe-trigger-web-snapshot-update.sh

```diff
@@ -53,9 +53,9 @@ execute () {
   curl -H "Authorization: token $GITHUB_TOKEN" \
       --request POST \
       --data "{ \"ref\": \"${BRANCH}\", \"inputs\": { \"snapshot_id\": \"${SNAPSHOT_ID}\" } }" \
-      https://api.github.com/repos/act-now-coalition/covid-projections/actions/workflows/update-snapshot.yml/dispatches
+      https://api.github.com/repos/act-now-coalition/covid-act-now-website/actions/workflows/update-snapshot.yml/dispatches
```

Comment:
This is the only breaking/necessary change here, the rest would redirect automatically, but I figured I'd clean them up while I was here